addons/dark-www/experimental_scratchwww.css

```
@@ -809,3 +809,12 @@ p {
      of the document inside, it gets an opaque backgound. */
   color-scheme: light;
 }
+
+/* Fix the accent color of checkboxes and radio buttons */
+label {
+  accent-color: var(--darkWww-button);
+}
+
+input {
+  accent-color: #129804;
+}
```

There's no reason to apply `accent-color` to both `input` and `label` elements.
```suggestion
input {
  accent-color: var(--darkWww-button);
}
```

---

Shouldn't this be `body:not(.sa-body-editor) input`, at least?